.asf.yaml

```diff
@@ -28,7 +28,8 @@ github:
   del_branch_on_merge: true
   #labels:
   enabled_merge_buttons:
-    merge:  false
+     # TODO: disable it after common merged
+    merge:  true
```

Comment:
seems we don't need to change in this commit

Reply:
> seems we don't need to change in this commit

Since another PR is blocked now